testing/run_tests.py

```diff
@@ -505,7 +505,8 @@ def make_test(name, flags=None, extra_env=None):
     # flutter_desktop_darwin_unittests uses global state that isn't handled
     # correctly by gtest-parallel.
     # https://github.com/flutter/flutter/issues/104789
-    if not os.path.basename(build_dir).startswith('host_debug'):
+    variant = os.path.basename(build_dir)
+    if not variant.startswith('host_debug') and 'arm64' not in variant:
       # Test is disabled for flaking in debug runs:
       # https://github.com/flutter/flutter/issues/127441
       run_engine_executable(
```

Comment:
So these are effectively not run by CI anymore right?

Reply:
Yup. I don't have time to do this, but the right thing to do would be to split off the desktop unit tests to continue running on an intel machine.

Reply:
Or make them run on arm64.  Is there an issue for why they can't run on arm64?

Reply:
I ran them locally on an arm64 machine in debug builds (without gtest-parallel) 5 times and they seem to work.

Reply:
I reproduced the error by forking this PR (weird the old result wasn't still available): https://github.com/flutter/flutter/issues/157205

Reply:
I have a fix in review for these tests: https://github.com/flutter/engine/pull/55990

Reply:
@gaaclarke I see that PR landed. Is it possible to remove one or both of these guards now?

Reply:
@zanderso It passed, you are good to rip out that change to run_tests.py